.github/workflows/pr-check.yml

```diff
@@ -4,7 +4,14 @@ name: hdf5 dev CI
 on:
   pull_request:
     branches: [ develop, hdf5_1_12, hdf5_1_10, hdf5_1_8 ]
-
+    paths-ignore:
+    - '.github/**'
+    - 'doc/**'
+    - 'release_docs/**'
+    - 'ACKNOWLEDGEMENTS'
+    - 'COPYING**'
+    - '**.md'
```

Comment:
Nothing new here - these were copied from main.yml

Reply:
Will it skip spelling tests on *.md files?

Reply:
That's a good question. Let me check to see if we can always run the spelling checks but not the code checks.

Reply:
spelling, format and code checks are all separate actions. Code and format are split into two actions - one for push and one for pull-requests